src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -380,10 +380,15 @@ protected virtual void OnUnprepared()
         /// Starts or resumes playback.
         /// </summary>
         /// <remarks>
-        /// The player must be in the <see cref="PlayerState.Ready"/> or <see cref="PlayerState.Paused"/> state.
+        /// Sound can be mixed with other sounds if you don't control the stream focus using <see cref="ApplyAudioStreamPolicy"/>.<br/>
+        /// Before Tizen 5.0, The player must be in the <see cref="PlayerState.Ready"/> or <see cref="PlayerState.Paused"/> state.
```

Comment:
How about belows?

```
/// Before Tizen 5.0, The player must be in the <see cref="PlayerState.Ready"/> or <see cref="PlayerState.Paused"/> state.
/// It has no effect if the player is already in the <see cref="PlayerState.Playing"/> state.<br/>
/// Since Tizen 5.0, The player must be in the <see cref="PlayerState.Ready"/>, <see cref="PlayerState.Playing"/>,
/// or <see cref="PlayerState.Paused"/> state.<br/>
```
->
```
/// The player must be in the <see cref="PlayerState.Ready"/> or <see cref="PlayerState.Paused"/> state.
/// Since API5, The player can be in the <see cref="PlayerState.Playing"/> state also, but it has no effect.<br/>
```